client/src/Forms/DefineForm.js

```diff
@@ -56,6 +60,8 @@ export default class DefineForm extends React.Component {
     };
 
     const respondToFetch = (response) => {
+      this.setState(defaultFieldValues);
+
```

Comment:
might be nice to not clear the form when submission fails, so they can try again without losing their work

Reply:
Good catch!